chat-plugins/info.js

```diff
@@ -552,6 +556,25 @@ var commands = exports.commands = {
 							}
 						}
 					}
+					if (Object.count(searches[search], true) > 1) {
```

Comment:
It might be better to define this as a function earlier so it can be called in `recover:` and `priority:` as well.